pf9/support/generate_bundle.py

```diff
@@ -0,0 +1,201 @@
+import os
+import sys
+import click
+import getpass
+import shlex
+import subprocess
+import requests
+import json
+from fabric import Connection, Config
+import paramiko.ssh_exception
+import socket
+import invoke.exceptions
+import time
+from progress.bar import Bar
+from pf9.exceptions import DUCommFailure, CLIException, UserAuthFailure
+from pf9.modules.express import Get
+from pf9.modules.util import Utils, Logger, Pf9ExpVersion
+
+class Log_Bundle:
+
+    def __init__(self,error_msg="none"):
+        self.error_msg = error_msg
+
+    def get_uuid_from_resmgr(self, du_url, host_ip, headers):
+        #Based on the host_ip this function would pull the host UUID from resmgr.
+        try:
+            api_endpoint = "resmgr/v1/hosts"
+            pf9_response = requests.get("{}/{}".format(du_url, api_endpoint),
+                                        headers=headers)
+            if pf9_response.status_code != 200:
+                return None
+            # parse resmgr response
+            json_response = json.loads(pf9_response.text)
```

Comment:
The `Response` object has a `json()` method that you can use directly instead of parsing the text attribute.
```
hosts = pf9_response.json()
```